notify/email/email.go

```diff
@@ -76,8 +77,21 @@ func (c *NotifyConfig) SendMail(subject string, message string) error {
 	}
 	body += "\r\n" + message
 
+	// support "," and ";"
+	split := func(r rune) bool {
+		return r == ';' || r == ','
+	}
+
+	recipients := strings.FieldsFunc(c.To, split)
+
 	auth := smtp.PlainAuth("", c.User, c.Pass, host)
 
+	if !c.SSL {
+		msg := []byte(body)
+		err = smtp.SendMail(c.Server, auth, c.User, recipients, msg)
```

Comment:
According to the comments, SendMail can switch to TLS if possible, can we just use it to support TLS and pure TCP at the same time?